opencensus/proto/agent/common/v1/common.proto

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// NOTE: This proto is experimental and is subject to change at this point.
+// Please do not use it at the moment.
+
+package opencensus.proto.agent.common.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.agent.common.v1";
+option java_outer_classname = "CommonProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/agentproto/commonproto/v1";
```

Comment:
Maybe consider:
github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1?

proto is already encoded in the "opencensus-proto/gen-go"?

@rakyll @odeke-em any opinion on this?

Reply:
v1 looks good.

Reply:
Sounds good, I would prefer to leave the name as-is, since in other protos the Go package names are similar, e.g:
["github.com/census-instrumentation/opencensus-proto/gen-go/traceproto"](https://github.com/census-instrumentation/opencensus-proto/blob/master/opencensus/proto/trace/trace.proto#L26)

Reply:
I think we should change all the package including the one you showed:
"github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
"github.com/census-instrumentation/opencensus-proto/gen-go/agent/trace/v1"

Reply:
Filed https://github.com/census-instrumentation/opencensus-proto/issues/82 to keep track of this.

Reply:
>@rakyll @odeke-em any opinion on this?

LGTM!